### DISCUSSION TITLE
MCCGQ14LM afected? (disconnects)

### DISCUSSION THREAD

**Discussion by ricargr** [Koenkk/zigbee2mqtt#10771] [Devices]
Hi, I have been reading this; 
https://www.zigbee2mqtt.io/devices/MCCGQ11LM.html#troubleshooting-device-stops-sending-messages-disconnects-from-network

I see in the description that nothing is said for the MCCGQ14LM, but can anyone confirm if this model is also affected?
I ask because in this page https://community.hubitat.com/t/xiaomi-aqara-devices-pairing-keeping-them-connected/623 references all the aqare devices, but this is from 2018.

Thank you

**Comment by c0mplex1** [comment]
Hi,

I use a combination of both MCCGQ11LM and MCCGQ14LM without any issues. No drops or pairing issues.
I use the latest firmware release in my Slae.sh CC2652RB stick.

**Reply by ricargr** [reply]
Thanks for the info, how much life time do you have with the batteries?

**Reply by c0mplex1** [reply]
I bought the Q11LM in January last Year and the Q14LM in November.
I thought the lifetime is about 2 Year.
So far, I haven't had to replace batteries yet.
I use them frequently, I think. Both switches detect 30-40 times a day door activities (Open/Close)

**Reply by ricargr** [reply]
Nice, thanks

**Comment by Shred99** [comment]
I have quite a number of MCCGQ11LM sensors and have recently added a few of the new E1 / MCCGQ14LM models.  Some of my sensors are 20-30m from the house and this is really pushing the limits, so I sometimes experience issues with the older MCCGQ11LM sensors dropping off the network and needing to either be moved close to a repeater or re-paired.

So far, my experience is that the MCCGQ14LM sensors do not suffer from the problems with dropping off the network.  Unlike the earlier Aqara devices, they seem to be quite happy to automatically use whichever repeater has the best signal a the time.  The older ones would sometimes change repeaters, but seem very reluctant and flakey in this regard.  If the signal isn't great, the new E1 sensors just stop working for a while, but they don't drop off the network and need re-pairing.  I haven't tested it extensively, but in one case, the E1 sensor doesn't seem to be as good at fucntioning over long distances.  I replaced an old sensor with a new one on a gate that is about 30m from the house.  The old sensor was working reasonably well, but the new E1 sensor seems a bit patchy.  It might be just a once off sensor that is a bit low on transmit power or has an aerial that's not quite tuned to resonate at the right frequency.  I won't know until I start using more of the E1 sensors over long distance links.

**Reply by ricargr** [reply]
Thanks for the detailed info. Can you confirm whats your zigbee router and firmware?

**Comment by Shred99** [comment]
There are two ZZH! routers, both running the current router firmware from this site.

The coordinator is a Texas Instruments LAUNCHXL-CC1352P-2, running the December 2021 coordinator firmware.
